nasl/nasl_misc_funcs.c

```diff
@@ -243,6 +244,7 @@ nasl_end_denial (lex_ctxt *lexic)
   int to = lexic->recv_timeout;
   struct script_infos *script_infos = lexic->script_infos;
   tree_cell *retc = NULL;
+  char bogus_data[64];
```

Comment:
I'm not really deep in C but isn't this limiting the amount of chars in that string which also means that the message could be stripped if the "vendor" string is really long?

Reply:
Therefore I leave the string at the end of the message. snprintf() will truncate the vendor string and add the '\0' char at the end. Do you think it would be necessary a bigger/dynamic buffer for the vendor?

Reply:
solved with https://github.com/greenbone/openvas-scanner/pull/665/commits/b81c402492b0b4bf72f038bcb6affdbdb1e0700d

Reply:
Thanks, that looks good. :+1:

Approving the text change but a full code review and a merge needs to be done by a @greenbone/gvm-dev member.